package.json

```diff
@@ -4,13 +4,19 @@
   "repository": "git@github.com:strahius/scopa.git",
   "license": "MIT",
   "workspaces": [
-    "packages/*"
+    "packages/shared",
+    "packages/server",
+    "packages/client"
   ],
   "scripts": {
     "client": "yarn workspace client start",
+    "heroku-client": "yarn workspace client heroku-start",
     "server": "yarn workspace server start",
+    "heroku-server": "yarn workspace server heroku-start",
     "shared": "yarn workspace shared start",
-    "start": "concurrently --kill-others-on-fail \"yarn server\" \"yarn shared\" \"yarn client\""
+    "start": "concurrently --kill-others-on-fail \"yarn server\" \"yarn shared\" \"yarn client\"",
+    "heroku-start": "concurrently --kill-others-on-fail \"yarn heroku-server\" \"yarn shared\" \"yarn heroku-client\"",
```

Comment:
I think here you should run only server, everything else should just build in the next line and that's it